targets/targets.json

```diff
@@ -2885,6 +2885,82 @@
         ],
         "device_name": "STM32H747XIHx"
     },
+    "PORTENTA_H7": {
+        "public": false,
+        "inherits": ["MCU_STM32H747xI"],
+        "config": {
+            "hse_value": {
+                "help": "HSE default value is 25MHz in HAL",
+                "value": "25000000",
+                "macro_name": "HSE_VALUE"
+            },
+            "lse_bypass": {
+                "help": "1 to use an oscillator (not a crystal) on 32k LSE",
+                "value": "1"
+            },
+            "usb_speed": {
+                "help": "USE_USB_OTG_FS or USE_USB_OTG_HS or USE_USB_HS_IN_FS",
+                "value": "USE_USB_OTG_HS"
+            }
+        },
+        "macros_add": [
+            "MBEDTLS_FS_IO",
+            "MBEDTLS_SHA1_C",
+            "CM4_BOOT_BY_APPLICATION",
+            "QSPI_NO_SAMPLE_SHIFT"
+        ],
+        "detect_code": ["0813"],
+        "components_add": [
+            "QSPIF",
+            "WHD",
+            "4343W",
+            "CYW43XXX"
+        ],
+        "extra_labels_add": [
+            "CORDIO"
+        ],
+        "features": [
+            "BLE"
+        ],
+        "device_has_add": [
+            "USBDEVICE",
+            "EMAC",
+            "QSPI"
+        ],
+        "overrides": {
+            "clock_source": "USE_PLL_HSE_EXTC",
+            "lse_available": 1,
+            "lpticker_delay_ticks": 0,
+            "network-default-interface-type": "ETHERNET"
```

Comment:
I don't see any Ethernet on the board...
Should be WIFI ? or CELLULAR ?

Reply:
The board has both an Ethernet phy mounted (exported like in the https://store.arduino.cc/portenta-vision-shield) and a WiFi module (but the fw lives in a user-selected filesystem so it's not useable "out of the box" in mbed).
I'd keep ETHERNET as default interface since it's zero config but no problem in changing it.

Reply:
Maybe several targets can be created depending on the several HW configuration that can exist ?